shared/src/common/enums/exceptions/exception-message.enum.ts

```diff
@@ -26,6 +26,10 @@ enum ExceptionMessage {
   TASK_COMPLETED = 'Task is already completed and cannot be updated.',
   USER_DETAILS_NOT_FOUND = 'User was not found.',
   FILE_TOO_BIG = 'This file is too large to upload. The maximum supported file size is: 1MB',
+  UNABLE_TO_PERFORM_BILLING_OPERATION = 'Unable to perform billing operation',
```

Comment:
```suggestion
  UNABLE_TO_PERFORM_BILLING_OPERATION = 'Unable to perform billing operation.',
```
Please, add for others also